src/installer/managed/Microsoft.NET.HostModel/MachO/CodeSignature.cs

```diff
@@ -0,0 +1,124 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO.MemoryMappedFiles;
+using System.Linq;
+
+namespace Microsoft.NET.HostModel.MachO;
+
+internal class CodeSignature
+{
+    private (EmbeddedSignatureHeader Header, long ptr) embeddedSignature;
+    private (CodeDirectoryHeader Header, long ptr) codeDirectory;
+    private (byte[] identifier, long ptr) identifierPtr;
+    private (byte[], long ptr) cdHashes;
```

Comment:
nit, can give first item a name (better than Item1 😅)
```suggestion
    private (byte[] blob, long ptr) cdHashes;
```